angular-translate/angular-translate.d.ts

```diff
@@ -72,7 +72,7 @@ declare namespace angular.translate {
     }
 
     interface ITranslateProvider extends angular.IServiceProvider {
-        translations(): ITranslationTable;
+        translations(key?: string): ITranslationTable;
         translations(key: string, translationTable: ITranslationTable): ITranslateProvider;
```

Comment:
the return type for this one should be `this` and not `ITranslateProvider ` see https://github.com/angular-translate/angular-translate/blob/master/src/service/translate.js#L251